lms/static/js/fixtures/edxnotes/edxnotes.html

```diff
@@ -1,7 +1,7 @@
 <section class="container">
   <div class="wrapper-student-notes">
     <div class="student-notes">
-
+        <main id="main" aria-label="Content" tabindex="-1">
```

Comment:
Many of those template files are a mess and don't have proper indentation. If I fix the indentation it kind of blurs the real work you guys are reviewing. These indentation fixes can come in a later PR. This comment applies for related comments below.

I will ensure the open/close tags I've added are on the same line though, as we discussed.